### PR TITLE
feat: add ready retry interval to config

### PR DESCRIPTION
**Pull request by v-stickykeys**
# Ready Retry Interval

## Description

Allows us to configure the interval for retrying requests marked as ready. "Retrying" means we emit an anchor event that these requests should be processed by. (Note: This is for a case in which a request marked as ready has not gotten processed yet, which can happen for example if the currently running workers are at capacity. So the next worker that starts will know to pick up the retried request if it can.)

## How Has This Been Tested?

TODO

## Notes

Related to an infra change https://github.com/3box/ceramic-infra/pull/634

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,6 +270,12 @@ In order to run tests with coverage run:
 
 ```npm run coverage```
 
+## Contributing
+
+### Config
+
+Values in the config files get their types auto-generated by [node-config-ts](https://github.com/tusharmath/node-config-ts) when running `npm run postinstall`.
+
 ## Maintainers
```
Comment (smrz2001) [inline README.md:279]: Can we finally change this lol? It bugs me everytime I come across it. I'd say @stbrody and @stephhuynh18 should probably be here instead.

**Review by stephhuynh18 — APPROVED** [review]
LGTM but i have no idea why the code coverage is failing

File: README.md (hunk 1 above)
Comment (v-stickykeys) [reply README.md:279]: haha yeah I will finalize a pr with changes for this

Comment (v-stickykeys) [reply README.md:279]: see https://github.com/ceramicnetwork/ceramic-anchor-service/pull/711

**Pull request merged**